HoaryFox/Member/FrameBreps.cs

```diff
@@ -156,11 +156,30 @@ public List<Brep> Frame(StbFrame frameData)
                     default:
                         throw new ArgumentOutOfRangeException();
                 }
-
-                brep.AddRange(CreateBreps.FromEndPoint(_stbData, nodeStart, nodeEnd, height, width, rotate, shapeType,  frameData.FrameType));
+                
+                var shapeInfo = new ShapeInfo(nodeStart, nodeEnd, height, width, rotate);
+                brep.AddRange(CreateBreps.FromEndPoint(_stbData, shapeInfo, shapeType, frameData.FrameType));
             }
 
             return brep;
         }
     }
+        
+    public struct ShapeInfo
```

Comment:
Codacy found an issue: [Implement 'IEquatable<T>' in value type 'ShapeInfo'.](https://app.codacy.com/gh/hrntsm/HoaryFox/pullRequest?prid=6416401)